src/Workspaces/Core/Portable/Rename/RenameLocation.ReferenceProcessing.cs

```diff
@@ -65,7 +65,7 @@ internal static class ReferenceProcessing
                 // backing field.
                 if (symbol.Kind == SymbolKind.Parameter)
                 {
-                    if (symbol.ContainingSymbol.Kind == SymbolKind.Method)
+                    if (symbol.ContainingSymbol?.Kind == SymbolKind.Method)
```

Comment:
Same comment.

Reply:
For this I can also just use `IsKind` which is allowed on `null`